distribution/openrct2.d.ts

```diff
@@ -2705,6 +2705,7 @@ declare global {
         start(): void;
         stop(): void;
         reset(): void;
+		isEnabled(): boolean;
```

Comment:
Use spaces instead of tabs:
```suggestion
        isEnabled(): boolean;
```